src/main/java/com/minecolonies/coremod/colony/buildings/BuildingMiner.java

```diff
@@ -30,6 +30,7 @@
 import java.util.HashMap;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.HashMap'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)